sal_interfaces/MTM1M3/MTM1M3_Events.xml

```diff
@@ -3410,6 +3410,13 @@
     <item>
       <EFDB_Name>motionState</EFDB_Name>
       <Description>Hardpoint Actuator State motion State</Description>
+      <Enumeration>
+          Standby=0,
+          Chasing=1,
+          Stepping=2,
+          QuickPositioning=3,
+          FinePositiong=4
+      </Enumeration>
```

Comment:
Please check the generated enum values in the IDL file (e.g. as generated with run make_idl_files.py MTM1M3 or I can generate it and send it to you if you prefer). I personally find the names for item-specific enums unsatisfactory so I always use global enums.